gettsim/tests/test_wohngeld.py

```diff
@@ -78,15 +79,57 @@ def test_wg(input_data, year, column):
     assert_series_equal(result[column], year_data[column])
 
 
+def eink_st_m_tu_from_data(eink_st_m, tu_id):
+    return eink_st_m.groupby(tu_id).sum()
+
+
+MAX_HH_SIZE = 12
+
+
 @pytest.fixture(scope="module")
-def input_data_2():
-    return pd.read_csv(ROOT_DIR / "tests" / "test_data" / "test_dfs_wg2.csv")
+def input_data_households():
+    df = pd.DataFrame(
+        data={
+            "p_id": 0,
+            "hh_id": np.arange(MAX_HH_SIZE + 1).repeat(np.arange(MAX_HH_SIZE + 1)),
+            "tu_id": np.arange(MAX_HH_SIZE + 1).repeat(np.arange(MAX_HH_SIZE + 1)),
+            "kind": False,
+            "kaltmiete_m_hh": 200,
+            "alleinerziehend": False,
+            "alter": 30,
+            "immobilie_baujahr_hh": 1970,
+            "kindergeld_anspruch": False,
+            "bruttolohn_m": 0,
+            "ges_rente_m": 0,
+            "_ertragsanteil": 0,
+            "elterngeld_m": 0,
+            "mietstufe": 0,
+            "arbeitsl_geld_m": 0,
+            "sonstig_eink_m": 0,
+            "unterhaltsvors_m": 0,
+            "brutto_eink_1": 0,
+            "brutto_eink_4": 0,
+            "brutto_eink_5": 0,
+            "brutto_eink_6": 0,
+            "eink_st_tu": 0,
+            "rentenv_beitr_m": 0,
+            "ges_krankenv_beitr_m": 0,
+            "behinderungsgrad": 0,
+        },
+        index=range(int((MAX_HH_SIZE * (MAX_HH_SIZE + 1)) / 2)),
```

Comment:
Maybe obvious for a mathematician :)

```suggestion
        index=range(np.sum(range(MAX_HH_SIZE + 1))),
```

Reply:
Or is the index even necessary after you expanded the values in the dictionary?

Reply:
Deleted explicit index!